.travis.yml

```diff
@@ -6,10 +6,9 @@ cache:
   directories:
   - node_modules
 before_install:
-- npm install -g yarn gulp bower
+- npm install -g yarn gulp
 install:
-- yarn install
-- bower install -F
+- yarn install --force
```

Comment:
Thats a good question, travis 💩's if its absent, I would say that force isn't so bad, just fetches deps fresh! https://yarnpkg.com/en/docs/cli/install#toc-yarn-install-force